blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -209,6 +242,29 @@ def Generate(self):
             self.append(item)
             return
 
+        # Generic (dis)connect
+        # LE devices do not appear to expose certain properties like uuids until connect to at least once.
+        # show generic connect for these as we will not show any way to connect otherwise.
+        device_uuids = self.SelectedDevice['UUIDs']
+        show_generic_connect = False if device_uuids else True
```

Comment:
`True if not device_uuids else False` is just `not device_uuids` or am I missing something?